nixarr/sonarr/settings-sync/default.nix

```diff
@@ -0,0 +1,173 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  inherit
+    (lib)
+    types
+    mkOption
+    getExe
+    mkIf
+    recursiveUpdate
+    ;
+
+  inherit
+    (pkgs.writers)
+    writeJSON
+    writePython3Bin
+    ;
+
+  nixarr = config.nixarr;
+  globals = config.util-nixarr.globals;
+  cfg = nixarr.sonarr.settings-sync;
+
+  nixarr-utils = import ../../lib/utils.nix {inherit pkgs lib config;};
+  inherit (nixarr-utils) arrCfgType;
+
+  nixarr-py = import ../../lib/nixarr-py {inherit pkgs lib config;};
+
+  show-schemas = writePython3Bin "nixarr-show-sonarr-schemas" {
+    libraries = [nixarr-py];
+    flakeIgnore = [
+      "E501" # Line too long
+    ];
+  } (builtins.readFile ./show_schemas.py);
+
+  sync-settings = writePython3Bin "nixarr-sync-sonarr-settings" {
+    libraries = [nixarr-py];
+    flakeIgnore = [
+      "E501" # Line too long
+    ];
+  } (builtins.readFile ./sync_settings.py);
+
+  downloadClientConfigType = types.submodule {
+    freeformType = arrCfgType;
+    options = {
+      name = mkOption {
+        type = types.str;
+        description = ''
+          The name Sonarr uses for this download client.
+          Note that names must be unique among *all download clients*, *ignoring case*.
+        '';
+      };
+      implementation = mkOption {
+        type = types.str;
+        description = ''
+          The implementation name of the download client in Sonarr. This is used
+          to find the default configuration when adding a new download client, and
+          must match the existing download client's implementation name when
+          overwriting an existing download client.
+        '';
+      };
+      enable = mkOption {
```

Comment:
I'm surprised this is necessary - is the default value in the schema not `true`?